docs/GettingStarted/DockerComposeSetup.md

```diff
@@ -80,12 +89,11 @@ DB_URL="mysql://YOUR_USER:YOUR_PASSWORD@YOUR_IP:YOUR_PORT/lake?charset=utf8mb4&p
 #      collation-server=utf8mb4_bin
 ```
 
-
-5. Final step: `docker compose up -d`
+6. Final step: `docker compose up -d`
 
 ### Can I use an external Grafana instead of running Grafana in Docker?
```

Comment:
I think it's missing two steps:

1. How do users connect their own Grafana to our database?
2. How do they import JSON dashboards to their Grafana instance?